README.md

```diff
@@ -0,0 +1,37 @@
+# GOB-Airflow
+
+GOB Airflow orchestrates GOB data pipelines.
+
+# Infrastructure
+
+A running [GOB infrastructure](https://github.com/Amsterdam/GOB-Infra)
+is required to run this component.
+
+You might want to stop GOB-Workflow however to avoid conflicts with queue's handling.
+
+# Docker
+
+## Requirements
+
+* docker-compose >= 1.17
+* docker ce >= 18.03
+
+## Run
+
+```bash
```

Comment:
Include some explanation of the reason for this docker. Does it run airflow or does it do something else?